src/scene/export.ts

```diff
@@ -102,6 +103,19 @@ export const exportToSvg = async (
       console.error(error);
     }
   }
+
+  const dummySvg = document.createElementNS(SVG_NS, "svg");
+  const rsvg = rough.svg(dummySvg);
+  elements
+    .filter((el) => !el.isDeleted)
+    .forEach((element) => {
+      try {
+        generateElementShape(element, rsvg.generator);
```

Comment:
This will also call `generateElementShape` for `images`, `text` as well, is that intended?

Reply:
Does it cause any issue? I see that it does not actually generate anything, it just sets it in `shapeCache`